apps/src/templates/teacherDashboard/teacherSectionsRedux.ts

```diff
@@ -259,7 +259,8 @@ const sectionSlice = createSlice({
             Object.keys(section).forEach(key => {
               if (
                 section[key as keyof Section] === undefined &&
-                prevSection[key as keyof Section] !== undefined
+                prevSection[key as keyof Section] !== undefined &&
+                !window.location.pathname.includes('/teacher_dashboard/home')
```

Comment:
I don't love basing logic on the URL. If there's another solution using props or arguments, I think that would be much more resilient, but understand if this is by far the easiest way.